model/atmosphere/subgrid_scale_physics/microphysics/src/icon4py/model/atmosphere/subgrid_scale_physics/microphysics/single_moment_six_class_gscp_graupel.py

```diff
@@ -0,0 +1,2915 @@
+    ice_stickeff_min: wpfloat = 0.01
+    #: Constant in v-qi ice terminal velocity-mixing ratio relationship, see eq. 5.169. Originally defined as tune_zvz0i in mo_tuning_nwp_config.f90 in ICON.
+    ice_v0: wpfloat = 1.25
+    #: Exponent of the density factor in ice terminal velocity equation to account for density (air thermodynamic state) change. Originally defined as tune_icesedi_exp in mo_tuning_nwp_config.f90 in ICON. Default value is 0.33.
+    ice_sedi_density_factor_exp: wpfloat = 0.3
+    #: Constant in v-D snow terminal velocity-Diameter relationship, see eqs. 5.57 (depricated after COSMO 3.0) and unnumbered eq. (v = 25 D^0.5) below eq. 5.159. Originally defined as tune_v0snow in mo_tuning_nwp_config.f90 in ICON.
+    snow_v0: wpfloat = 20.0
+    #: mu exponential factor in gamma distribution of rain particles. Originally defined as mu_rain in mo_atm_phy_nwp_config.f90 in ICON.
+    rain_mu: wpfloat = 0.0
+    #: Interception parameter in gamma distribution of rain particles. Originally defined as rain_n0_factor in mo_atm_phy_nwp_config.f90 in ICON.
+    rain_n0: wpfloat = 1.0
+
+
+# TODO (Chia Rui): Refactor these constants when the restriction of FrozenNameSpace for compile-time constants is lifted.
+class SingleMomentSixClassIconGraupelParams(FrozenNamespace):
+    """
+    Contains numerical, physical, and empirical constants for the ICON graupel scheme.
+
+    These constants are not configurable from namelists in ICON.
+    If users want to tune the model for better results in specific cases, you may need to change the hard coded constants here.
+    All the equations can be found in A Description of the Nonhydrostatic Regional COSMO-Model Part II Physical Parameterizations.
+    """
+
+    #: threshold temperature for heterogeneous freezing of raindrops. Originally expressed as trfrz in ICON.
+    threshold_freeze_temperature: wpfloat = 271.15
+    #: FR: 1. coefficient for immersion raindrop freezing: alpha_if, see eq. 5.168. Originally expressed as crfrz1 in ICON.
+    coeff_rain_freeze1_mode: wpfloat = 9.95e-5
+    #: FR: 2. coefficient for immersion raindrop freezing: a_if, see eq. 5.168. Originally expressed as crfrz2 in ICON.
+    coeff_rain_freeze2_mode: wpfloat = 0.66
+    #: temperature for hom. freezing of cloud water. Originally expressed as thn in ICON.
+    homogeneous_freeze_temperature: wpfloat = 236.15
+    #: threshold temperature for mixed-phase cloud freezing of cloud drops (Forbes 2012, Forbes & Ahlgrimm 2014), see eq. 5.166. Originally expressed as tmix in ICON.
+    threshold_freeze_temperature_mixedphase: wpfloat = 250.15
+    #: threshold for lowest detectable mixing ratios.
+    qmin: wpfloat = 1.0e-15
+    #: a small number for cloud existence criterion.
+    eps: wpfloat = 1.0e-15
```

Comment:
From my understanding, I can simply use `DBL_EPS` for `eps` because it is only a small number that prevent division-by-zero that should be system dependent. For `qmin`, I would need that precise value. A different value will greatly alter the result based on my experience with different microphysics.